package.json

```diff
@@ -20,7 +20,7 @@
     "dist"
   ],
   "engines": {
-    "node": "~16"
+    "node": ">=16"
```

Comment:
Because of the `adm-zip` I had to change the engine.
I think that we can leave it as it is, and also consider bumping the lambda@edge runtime to node 18 at some point.